index.html

```diff
@@ -0,0 +1,107 @@
+<html>
+  <head>
+    <title>Personal Portfolio Webpage</title>
+    <link rel="stylesheet" href="index.css"/>
+  </head>
+  <body>
+    <header>
+      <nav id="navbar">
+        <ul>
+          <li>
+            <a href="https://personal-portfolio.freecodecamp.rocks/#welcome-section">About</a>
+          </li>
+          <li>
+            <a href="https://personal-portfolio.freecodecamp.rocks/#welcome-section">Work</a>
+          </li>       
+          <li>
+            <a href="https://personal-portfolio.freecodecamp.rocks/#welcome-section">Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="welcome-section" class="welcome">
+        <h1>Hey I am Mimic</h1>
+        <p>a web developer</p>
+      </section>
+
+      <section id="projects" class="flow">
+        <h1>These are some of my projects</h1>
```

Comment:
This line looks beautiful but it could be better if it's  h2 for it not to be the main heading. also for semantics reasons